netsuitesdk/api/journal_entries.py

```diff
@@ -8,11 +8,33 @@
 
 
 class JournalEntries(ApiBase):
```

Comment:
Rather than copy-pasting this entire class into `adv_inter_company_journal_entries.py`, I made some refactoring to enable simple inheritance. I hope that's okay!